code/modules/chemistry/Chemistry-Reagents.dm

```diff
@@ -297,6 +298,13 @@ datum
 				return AD
 			return
 
+		proc/flush(var/mob/M, var/mult = 1, var/amount, var/list/flush_specific_reagents)
+			for (var/reagent_id in M.reagents.reagent_list)
+				if ((reagent_id != id) && ((reagent_id in flush_specific_reagents) || (!flush_specific_reagents)))//checks if there's a specific reagent list to flush or if it should flush all reagents.
```

Comment:
```suggestion
				if ((reagent_id != src.id) && ((reagent_id in flush_specific_reagents) || !flush_specific_reagents))//checks if there's a specific reagent list to flush or if it should flush all reagents.
```
style stuff, but makes it more readable imo

Reply:
sounds good to me!